examples/dws/cluster_secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    testing.upbound.io/example-name: example_dws_password
+  name: example-dws-password
```

Comment:
Isn't it better to use `example_dws_cluster_secret` and `example-dws-cluster-secret` ?